### ISSUE TITLE
Enable zooming and scaling for Paypal Button

### ISSUE THREAD

**Issue by youriwims** [mozilla/donate-wagtail#302]
### Issue: [Zooming and scaling must not be disabled](https://dequeuniversity.com/rules/axe/3.3/meta-viewport?application=msftAI)

**Element path:** `#zoid-paypal-button-2adaacf6ec > .zoid-outlet > iframe;meta[name="viewport"]`

**Snippet:** `<meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=no">`

**How to fix:** 
Fix any of the following:
- user-scalable=no on <meta> tag disables zooming on mobile devices

====

More Info:

Title: WCAG 1.4.4: Zooming and scaling must not be disabled (meta[name="viewport"])
Tags: Accessibility, WCAG 1.4.4, meta-viewport
Environment: Chrome version 77.0.3865.90
Comments: Not sure how simple removing this would be, but this is a critical a11y violation and is in need of some attention either way.

**Comment by nicklee** [comment]
@alanmoo This is a good point but is out of our control, the third party paypal javascript adds the iframe that contains the meta tag, meaning we can't change it.

**Comment by mmmavis** [comment]
Cannot fix due to the reason above. Closing.
